### DISCUSSION TITLE
Silicon training

### DISCUSSION THREAD

**Discussion by Mskater** [deepmodeling/deepmd-kit#721] [Q&A]
Hi,
Can someone please tell how to modify the water_se_a.json training and testing files (or other compatible files) to train and test the given silicon folder data?    I see the comment in the doc that says input files are "highly similar", but I'm not familiar enough with the details, as just starting out.
For instance, is there virial data given?  The water example in the doc says it has no virial data, so those options are set to zero.
What else is different?  
Thanks very much.

**Comment by njzjz** [comment]
We provide the documents for parameters here: https://deepmd.readthedocs.io/en/v1.3.3/train-input.html
